helm/templates/configmap.yaml

```
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.cvmsConfig.name }}
+  namespace: {{ default .Release.Namespace .Values.namespaceOverride | quote }}
+  labels:
+    {{- include "cvms.labels" . | nindent 4 }}
+    {{- with .Values.cvmsConfig.exteraLabels }}
```

yamllint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)